trapper/metrics/metric.py

```diff
@@ -19,31 +19,31 @@ class Metric(Registrable, metaclass=ABCMeta):
     compute score for that prediction.
```

Comment:
Can we split metric.py file into two, namely metric.py and jury.py. Then move the Jury class definiton to jury.py. Actually I made that comment before, but I guess it was missed (or I haven't seen the reply)